app/controllers/reviews_controller.rb

```diff
@@ -118,4 +119,39 @@ def check_review_period
 
     redirect_to root_path, alert: t('reviews.edit.errors.conference_out_of_range')
   end
+
+  def create_third_reject(review)
+    return if in_early_review_phase?
+
+    return unless review.weak_reject? || review.strong_reject?
+
+    session_reviews = review.session.early_reviews + review.session.final_reviews
+    other_reviews = session_reviews.reject { |r| r == review }
+
+    return if other_reviews.count != 1
+
+    other_review = other_reviews.first
+
+    return unless other_review.weak_reject? || other_review.strong_reject?
+
+    low_rating = Rating.find_by(title: 'rating.low.title')
+    resource_class.create!(
+      session: review.session,
+      recommendation: Recommendation.find_by(name: 'strong_reject'),
+      reviewer_id: 742,
```

Comment:
Reviewer id is attached to a specific instance of a conference. Using last year's won't work and might break something else.

Reply:
This also will only work in production which is bad. We need to do a find or create on a reviewer that we can identify. This can be a reviewer attached to a user in each environment with seeds that is an auto reviewer (by name or email).